src/lib/utils/interfaces/team.interface.ts

```diff
@@ -11,19 +12,34 @@ export interface CurrentTeam {
 }
 
 export interface TeamRepositoryMethods {
-  getDocument: (elem: TeamDocument) => any;
-  getTeams: () => Observable<TeamDocument[]>;
-  checkActiveTeam: (teamId: string) => Promise<boolean>;
-  clearTeams: () => Promise<any>;
-  setActiveTeam: (teamId: string) => Promise<void>;
-  activateInitialTeamWithWorkspace: () => Promise<string>;
-  bulkInsertData: (data: any) => Promise<void>;
-  getActiveTeam: () => Observable<TeamDocument>;
-  createTeam: (team: any) => Promise<void>;
-  modifyTeam: (teamId: string, data) => any;
+  getDocument?: (elem: TeamDocument) => any;
+  getTeams?: () => Observable<TeamDocument[]>;
+  checkActiveTeam?: (teamId: string) => Promise<boolean>;
+  clearTeams?: () => Promise<any>;
+  setActiveTeam?: (teamId: string) => Promise<void>;
+  activateInitialTeamWithWorkspace?: () => Promise<string>;
+  bulkInsertData?: (data: any) => Promise<void>;
+  getActiveTeam?: () => Observable<TeamDocument>;
+  createTeam?: (team: any) => Promise<void>;
+  modifyTeam?: (teamId: string, data) => any;
+  setOpenTeam?: (teamId: string) => any;
 }
 
 export interface TeamServiceMethods {
-  fetchTeams: (userId: string) => any;
-  createTeam: (team: TeamPostBody) => any;
+  fetchTeams?: (userId: string) => any;
+  createTeam?: (team: TeamPostBody) => any;
+
+  inviteMembersAtTeam?: (teamId: string, inviteBody: InviteBody) => unknown;
```

Comment:
is the return type unknown. Can we make it a boolean type in case of success or void.

Reply:
It is unknown, but sure it cant be void or boolean.